docs/standard/native-interop/best-practices.md

```diff
@@ -297,7 +315,7 @@ There are types in the C/C++ language that have latitude in how they are defined
 
 ### C/C++ `long`
 
-C/C++ `long` and C# `long` are not the same types. Using C# `long` to interop with C/C++ `long` is almost never correct.
+C/C++ `long` and C# `long` are not the same types. Using C# `long` to interop with C/C++ `long` is not correct unless you are only targeting Windows.
```

Comment:
Using C# long with C++ long on Windows is never correct. It's sometimes correct on non-Windows.

Reply:
C# long is always 64-bit. C++ long is 32bit on all Windows platforms. You probably want to say that using `int` for `long` is only correct on Windows.

Reply:
doh - yes. How does it look now.